packages/client/src/components/shared/TerminalResult.tsx

```diff
@@ -7,19 +7,31 @@ interface TerminalResultProps {
 }
 
 @observer
-export default class IndividualMetricView extends React.Component<TerminalResultProps> {
+export default class TerminalResult extends React.Component<TerminalResultProps> {
   render(): React.ReactNode {
     const { stackTrace, error, errors } = this.props.exception.details;
     return (
       <div className="terminal-canary">
         <div className="main-error">{error}</div>
         <div className="stacktrace">
-          <div className="stacktrace-label">Error</div>
-          {errors.map((error: string) => (
-            <div>{error}</div>
-          ))}
-          <div className="stacktrace-label">Stacktrace</div>
-          <pre>{stackTrace}</pre>
+            {errors ?
+                <div>
+                    <div className="stacktrace-label">Error</div>
+                    errors.map((error: string) => (
+                        <div>{error}</div>
+                    ))
+                    </div>
+                :
+                null
+            }
```

Comment:
This can be changed from a ternary statement to just using an AND operator. Something like:

`{errors &&
                (<div>
                    <div className="stacktrace-label">Error</div>
                    errors.map((error: string) => (
                        <div>{error}</div>
                    ))
                    </div>)
            }`


See: https://github.com/Nike-Inc/referee/blob/master/packages/client/src/components/config/MetricSummaries.tsx#L26

Reply:
fixed